akiraux/Akira
# More artboard code

## Summary / How this PR fixes the problem?
This PR implements custom CanvasItemSimple with uniform strokes (including hit testing), and better clipping functionality. The PR also uses that clipping to implement the expected artboard behavior.

I also used the new drawables for the hover effects in order to have them render in a better way

## Steps to Test
Skewed shapes should have uniform stroke sizes and be drawn correctly. Press G to create an artboard with objects, and the artboard should clip the children correctly.
